main.js

```diff
@@ -215,3 +215,34 @@ function checkRange() {
     updateButton.disabled = true;
   }
 }
+
+// new code for error message when guesses are outside the range
+var rangeErrorMessageOne = document.querySelector('.range-error-one');
+var rangeErrorMessageTwo = document.querySelector('.range-error-two');
+
+document.addEventListener('keyup', rangeErrorCheckOne);
+document.addEventListener('keyup', rangeErrorCheckTwo);
```

Comment:
I've started adding the keyup event listeners to the inputs themselves, rather than the document.